.travis.yml

```diff
@@ -36,14 +36,10 @@ env:
     - DOWNLOAD_CACHE=$HOME/download-cache
     - INSTALL_CACHE=$HOME/install-cache
     - JOBS=2
-    - KONG_TEST_PG_USER=postgres
-    - KONG_TEST_PG_DATABASE=travis
-    - KONG_TEST_CASSANDRA_KEYSPACE=kong_tests
-    - KONG_TEST_DB_UPDATE_PROPAGATION=1
   matrix:
-    - KONG_TEST_DATABASE=postgres
-    - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2
-    - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9
+    - KONG_TEST_DATABASE=postgres KONG_TEST_PG_DATABASE=travis KONG_TEST_PG_USER=postgres KONG_TEST_PG_DATABASE=travis
+    - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 KONG_TEST_CASSANDRA_KEYSPACE=kong_tests KONG_TEST_DB_UPDATE_PROPAGATION=1
+    - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 KONG_TEST_CASSANDRA_KEYSPACE=kong_tests KONG_TEST_DB_UPDATE_PROPAGATION=1
```

Comment:
I think I have suggested it before, but those variables should be added to `.ci/setup_env.sh` _or_ `.ci/run_tests.sh`. Since they are very specific to the matrix itself, and duplicated all over the place if we keep them here (especially when we start adding new rows to this build matrix).